style.css

```diff
@@ -4,7 +4,7 @@ Theme URI: https://github.com/asu-ke-web-services/UDS-WordPress-Theme
 Author: Arizona State University Knowledge Enterprise
 Author URI: https://research.asu.edu
 Description: Combination of Automattic´s _s theme and Bootstrap 4, customized and enhanced for ASU Web Standards 2.0
-Version: 0.53
+Version: 0.54
```

Comment:
I believe we prefer not to have version bumps in our actual feature branches, but rather to bump the version on the main branch (develop, in this case) **after** the merge. This is just in case a branch languishes...like this one did...and the main branch version has already been bumped as a result of some other PR being merged.